beamer/tests/agent/test_request.py

```diff
@@ -364,3 +364,17 @@ def test_agent_only_claim_once_after_restart(request_manager, token, agent, dire
                     sleeper.sleep(0.1)
                 except Timeout:
                     break
+
+
+def test_event_fetcher_logs(agent, direction):
```

Comment:
Nitpick: I would rename the test to better reflect what is being test, e.g. `test_event_fetcher_progress_without_events`.